### PR TITLE
Implemented skip countdown logic and changed the looks a bit

### PR DESCRIPTION
**Pull request by YaroslavKryvko**
@VerizonAdPlatforms/mobile-sdk-developers: Ready for review.

- Added skip countdown logic
- Added disabled state
- Reworked visuals a bit
screen shot 2019-01-09 at 16 11 49

screen shot 2019-01-09 at 16 25 32

[JIRA](https://jira.ops.aol.com/browse/OMSDK-2129)

### CODE REVIEW

File: controls/src/main/java/com/aol/mobile/sdk/controls/view/AdControlsView.java [hunk 1]
```diff
@@ -238,6 +239,12 @@ public void render(@NonNull ViewModel vm) {
         renderVisibility(vm.isPauseButtonVisible, pauseButton);
         renderVisibility(vm.isCloseButtonVisible, clickthroughClose);
         renderVisibility(vm.isSkipButtonVisible, skipButton);
+        String skipText = getResources().getString(R.string.skip);
+        if (vm.skipCountdown != null) {
```
Comment (AleksanderLevadniy) [inline controls/src/main/java/com/aol/mobile/sdk/controls/view/AdControlsView.java:245]: Rework to resource string with placeholders like [here](http://sergiodelamo.es/android-tips-how-to-use-placeholders-in-an-android-string-resource/)

**Review by AleksanderLevadniy — DISMISSED** [review]
Please, review comments

**Review by AndriiMoskvin — DISMISSED** [review]
Can the text inside be centered?

**Comment by YaroslavKryvko** [comment]
This right padding is left for ">" icon which will be added later

**Review by AleksanderLevadniy — APPROVED** [review]
👍 Waiting for arrow graphics version

**Pull request merged**